lib/smart_proxy_pulp_plugin/pulp_plugin.rb

```diff
@@ -1,9 +1,10 @@
 module PulpProxy
   class Plugin < ::Proxy::Plugin
     plugin "pulp", ::PulpProxy::VERSION
-    default_settings :pulp_url => 'https://localhost/pulp'
+    default_settings :pulp_url => 'https://localhost/pulp',
+                     :pulp_dir => '/var/lib/pulp'
```

Comment:
I would suggest using 'validate_readable' for 'pulp_dir' and all other filesystem paths (see example here: https://github.com/theforeman/smart-proxy/blob/develop/modules/dhcp_isc/dhcp_isc_plugin.rb#L10). This way invalid paths will be reported on startup and the module will be disabled.